pkg/operator/ceohelpers/common.go

```diff
@@ -178,14 +178,14 @@ func memberToURL(member *etcdserverpb.Member) (string, error) {
 	return member.PeerURLs[0], nil
 }
 
-func VotingMemberIPListSet(configMapLister corev1listers.ConfigMapNamespaceLister) (sets.String, error) {
-	etcdEndpointsConfigMap, err := configMapLister.Get("etcd-endpoints")
+func VotingMemberIPListSet(ctx context.Context, cli etcdcli.EtcdClient) (sets.String, error) {
+	members, err := cli.VotingMemberList(ctx)
```

Comment:
@tjungblu  is this correct ?

Reply:
just replicate what the member controller does in:
https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/etcdendpointscontroller/etcdendpointscontroller.go#L132-L136

Reply:
good catch 👍🏽